source/_integrations/synology_dsm.markdown

```diff
@@ -82,7 +82,7 @@ Entities reporting the internal temperature, status (as shown in Synology DSM) a
 
 ### Volume sensors
 
-Entities reporting status, total size (TB), used size (TB), % of volume used, average disk temperature and maximum disk temperature for each volume inside the NAS. By default the total size and maximum disk temperature sensors are disabled.
```

Comment:
The integration says code current claims it is exporting TB so we have a bit of a disagreement there...

https://github.com/home-assistant/core/blob/0dc1e7d1e64a0172c49a1114f811797283c12de7/homeassistant/components/synology_dsm/sensor.py#L192

To be clear, I'm not claiming you're wrong about the discrepancy in units, but it sounds like there may be a larger issue and just updating the docs here may not be enough.

Reply:
therefore the linked PR for code change 😉

Reply:
:facepalm: